### PR TITLE
ES: Add 7 Noticias (YouTube and Twitch)

### PR DESCRIPTION
**Pull request by joaquinito2048**

### CODE REVIEW

**Review by BellezaEmporium — DISMISSED** [review]
Watch out as Heroku might take it down whether they like it, that's basically what happened with my implementation.

File: streams/es.m3u [hunk 1]
```diff
@@ -5,6 +5,10 @@ https://directes-tv-int.ccma.cat/int/ngrp:324_web/master.m3u8
 https://pc-la7delarioja-hls.flumotion.com/playlist.m3u8
 #EXTINF:-1 tvg-id="7Noticias.es" status="online",7 Noticias (1080p)
 https://amg01573-7nn-7nnono-ono-pcdj3.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="7Noticias.es",7 Noticias (1080p) [Not 24/7]
+https://streamlink-iptv.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCCydt40S3mVi9YXQJjK8Nsw/live
```
Comment (freearhey) [inline streams/es.m3u:9]: image

File: streams/es.m3u [hunk 2]
```diff
@@ -5,6 +5,10 @@ https://directes-tv-int.ccma.cat/int/ngrp:324_web/master.m3u8
...
+#EXTINF:-1 tvg-id="7Noticias.es",7 Noticias (1080p)
+https://streamlink-iptv.herokuapp.com/iptv-query?streaming-ip=https://www.twitch.tv/7nntv
```
Comment (freearhey) [inline streams/es.m3u:11]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the links again.

**Comment by LaneSh4d0w** [comment]
As I've predicted, Heroku most certainly took it down.

File: streams/es.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/es.m3u:9]: http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCCydt40S3mVi9YXQJjK8Nsw/live

File: streams/es.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply streams/es.m3u:11]: http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/7nntv

**Comment by joaquinito2048** [comment]
@LaneSh4d0w @freearhey Approve and merge.

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — COMMENTED** [review]
Cannot test if links are online, server returns 502.

**Review by MapGuy11 — APPROVED** [review]

**Pull request merged**